lms/templates/certificates/valid.html

```diff
@@ -0,0 +1,150 @@
+                <span class="value">${certificate_id_number}</span>
+              </li>
+              <li class="item certificate--date">
+                <span class="label">${certificate_date_issued_title}</span>
+                <span class="value">${certificate_date_issued}</span>
+              </li>
+            </ul>
+          </div>
+        </article>
+      </section>
+      <hr class="divider /">
+      <aside class="content content--supplemental" role="complimentary"  id="validation-info">
+        <div class="supplemental__about">
+          <h2 class="title">${company_about_title}</h2>
+          <div class="copy">
+            <p>${company_about_description}</p>
+          </div>
+          <ul class="list list--actions">
+            <li class="item item--action">
+              <a class="action action--primary action--moreinfo" href="${company_about_url}">${company_about_urltext}</a>
+            </li>
+          </ul>
+        </div>
+        <div class="supplemental__how">
+          <h2 class="title">${certificate_verify_title}</h2>
+          <div class="copy">
+            <p>${certificate_verify_description}</p>
+          </div>
+          <ul class="list list--actions">
+            <li class="item item--action">
+              <a class="action action--moreinfo" href="${certificate_verify_url}">${certificate_verify_urltext}</a>
+            </li>
+          </ul>
+        </div>
+        <div class="supplemental__certificates">
+          <h2 class="title">${certificate_info_title}</h2>
+          <div class="copy">
+            <p>${certificate_info_description}</p>
+          </div>
+        </div>
+      </aside>
+    </div>
+    <hr class="divider /">
+    <div class="wrapper--footer">
+      <footer class="footer--app" role="contentinfo" id="company-info">
+        <div class="copyright">
+          <p>${copyright_text}</p>
+        </div>
+        <nav class="nav--footer">
+          <ul class="list list--legal">
+            <li class="nav__item">
+              <a class="action" href="${company_tos_url}"></a>
+            </li>
+            <li class="nav__item">
+              <a class="action" href="${company_privacy_url}">${company_privacy_urltext}</a>
+            </li>
+          </ul>
+          <ul class="list list--actions">
+            <li class="nav__item">
+              <a class="action" href="${company_courselist_url}">${company_courselist_urltext}</a>
+            </li>
+            <li class="nav__item">
+              <a class="action" href="${company_careers_url}">${company_careers_urltext}</a>
+            </li>
+            <li class="nav__item">
+              <a class="action" href="${company_contact_url}">${company_contact_urltext}</a>
+            </li>
+          </ul>
+        </nav>
+      </footer>
+    </div>
+  </div>
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
```

Comment:
I agree that it seems unnecessary to include JQuery here -- however this template is a near-duplicate of the one located at https://verify-test.edx.org/cert/dc75ee7bdfa5457e882466e85eca3ea3 -- as you noted, we'll be receiving an updated set of markup from UX as a deliverable for their next sprint.